cfme/infrastructure/virtual_machines.py

```diff
@@ -657,6 +692,17 @@ def delete_disk(self, filename=None, index=None):
     def num_disks(self):
         return len(self.disks)
 
+    def add_nw_adapter(self, name, vlan):
```

Comment:
I'm not a fan of the `nw` abbreviation here for network, especially mixed with the NetworkAdapter class names.

I also don't think we need a 'num_nw_adapters' property here, the caller can easily use a `len()` call.

Reply:
> I'm not a fan of the `nw` abbreviation here for network, especially mixed with the NetworkAdapter class names.

changed `nw > network`
> 
> I also don't think we need a 'num_nw_adapters' property here, the caller can easily use a `len()` call.
Its very useful while playing with reconfiguration. You can say just like `num_disks`; I tried to make it similar to `disk` operations. So it will be very easy to write network reconfiguration related tests.